client/flutter/lib/l10n/your_questions_answered.yaml

```
@@ -0,0 +1,35 @@
+schema_version: 1
```

another non-blocker, but consider using minor version?

```suggestion
schema_version: 1.0
```

---

IMO think these should be SOLELY breaking versions, and integers.  Everything else should be optional fields, like in protobufs.

---

in that case, I do propose that we add a `content_version` for eventually when we want to use caching. but that's definitely not for v1.0 and definitely not a blocker on this PR

---

Yes content_version should def be there

---

I will add content_version